redis/src/commands/cluster_scan.rs

```diff
@@ -430,19 +443,6 @@ where
                     retry_scan(&scan_state, &core, match_pattern, count, object_type).await?;
                 (from_redis_value(&retry.0?)?, retry.1)
             }
-            ErrorKind::BusyLoadingError | ErrorKind::TryAgain => (
```

Comment:
Can you add doc why only ErrorKind::IoError | ErrorKind::ClusterConnectionNotFound  are being handled?

Reply:
Please add a todo to fix it:
Err(err) => Err((OperationTarget::FanOut, err)),